js/components/notificationBar.js

```diff
@@ -115,6 +116,14 @@ class NotificationBar extends ImmutableComponent {
   }
 }
 
+class NotificationBarCaret extends ImmutableComponent {
```

Comment:
I prefer this way because then I can leave styles on the component itself. Otherwise, I'll have to create a new style for that and put it on the tab itself or inside a third file, and if I do it then I'm not taking advantage of Aphrodite.

Style and component should live together so I don't have to run several files to find a simple thing. Things we do such as `commonStyles.js` is just a hack we found to keep refactoring important stuff while common components aren't ready yet/too nested. Having both together makes regressions less likely. 

It also simplifies the code, so if for some reason Brad or other less code-experienced member/collaborator wants to change something for a given component, they'll need to go to into just one file.